Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM amazon/aws-cli
 
-RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl && \
+RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && \
```

Comment:
could this be passed in as an ENV ? in preference to altering the entrypoint.sh ?

Reply:
That sounds good
Probably can be done checking if the env var for the kubectl version is set, and in that case curl using that, else curl using the latest stable as it's now. Should be fairly straightforward

Reply:
This would be preferable, but I believe that the docker image is created from the GitHub Action call before reading ENVs, so passing an ENV before build isn't possible. Alternatively, an input arg can be defined in action.yml file for the action itself, but would still require a modified entrypoint to capture the value for the desired version to download.